module3/homework/inner_product/arithmeticAverage.cpp

```diff
@@ -0,0 +1,13 @@
+#include <cmath>
+#include <functional>
+#include <numeric>
+#include <vector>
+
+double ArithmeticAverage(std::vector<int>& v1, std::vector<int>& v2) {
+    double size = v1.size() + v2.size();
+    return std::inner_product(v1.begin(), v1.end(), v2.begin(), 0, std::plus<>(), std::plus<>()) / size;
```

Comment:
What do You think about using constant iterators? I think that You will not modify them in this exercise. Of course it isn't any issue. It's just good practice in my opinion.

Reply:
Tbh I consistently forget about constant operators

Reply:
And I would suggest to put them in the header,since it makes const variable easier to access in case you would like to modify them.